combat-trainer.lic

```diff
@@ -1055,15 +1055,15 @@ class TrainerProcess
     dead_count = DRRoom.dead_npcs.size
 
     EquipmentManager.instance.stow_weapon(game_state.weapon_name)
-    EquipmentManager.instance.wield_weapon(game_state.ambush_stun_weapon, 'Small Blunt')
+    EquipmentManager.instance.wield_weapon(@stun_weapon)
```

Comment:
I'll set this up with another instance variable and then try to think of a way to do it without adding another one.